src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts

```diff
@@ -89,6 +91,22 @@ export class InstructorSessionResultPageComponent implements OnInit {
         }, (resp2: any) => {
           this.statusMessageService.showErrorMessage(resp2.error.message);
         });
+
+        this.httpRequestService.get('/students', paramMap).subscribe((resp3: any) => {
```

Comment:
- `resp3` is not a good name.
- You can change `any` to a concrete type in `api-output.ts`.

Reply:
Sure.